core-api/src/test/java/com/sequenceiq/cloudbreak/validation/CredentialRequestValidatorTest.java

```diff
@@ -0,0 +1,157 @@
+package com.sequenceiq.cloudbreak.validation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import javax.validation.ConstraintValidatorContext;
+
+import org.junit.Test;
+
+import com.sequenceiq.cloudbreak.api.model.CredentialRequest;
+
+/**
+ * Created by gpapp on 6/14/18.
+ */
+public class CredentialRequestValidatorTest {
+
+    private static final String CREDENTIAL_REQUEST_KEYSTONE_VERSION_KEY = "keystoneVersion";
```

Comment:
Looks like already defined contants (on my phone)

Reply:
I redefined the constants in the tests to verify the contents of the constants as well.